src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,50 @@
 package mate.jdbc;
 
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        ManufacturerService manufacturerService = (ManufacturerService)
+                injector.getInstance(ManufacturerService.class);
+        Manufacturer audi = new Manufacturer("audi", "Italy");
+        Manufacturer volvo = new Manufacturer("volvo", "Germany");
+        Manufacturer reno = new Manufacturer("reno", "France");
+        manufacturerService.create(audi);
+        manufacturerService.create(volvo);
+        manufacturerService.create(reno);
+        System.out.println(manufacturerService.get(audi.getId()));
+        manufacturerService.delete(audi.getId());
+        Manufacturer manufacturerToUpdate = new Manufacturer(2L, "shkoda", "Japan");
+        manufacturerService.update(manufacturerToUpdate);
+        Manufacturer manufacturerWithIndex3 = manufacturerService.get(3L);
+        Manufacturer manufacturerToCompare = new Manufacturer(3L, "reno", "France");
```

Comment:
here in your example/test code let's not use hardcoded int literals as id parameter. I'd recommend creating and assigning the manufacturer to some variable first. And then you can retrieve from this variable an id field and pass it to the corresponding methods